syntax = "proto3";

package payment;


message Transaction_Proto {
  string stripe_id = 1;
  int32 user_id = 2;
  int32 amount = 3;
  string currency = 4;
  string status = 5;
  string description = 6;
  string transaction_date = 7;
  string payment_method = 8;
  
}



